src/Microsoft.SqlTools.ServiceLayer/QueryExecution/SpecialAction.cs

```diff
@@ -72,7 +72,7 @@ public bool ExpectYukonXMLShowPlan
         /// </summary>
         public void CombineSpecialAction(SpecialAction action)
         {
-            flags |= action.flags;
+            flags |= ((action?.flags) ?? ActionFlags.None);
```

Comment:
This just fixes an independent pre-existing null reference exception as we were derefencing action which was allowed to be passed in a null value.